content/blog/deploying-netlify-cms-on-aws/index.md

```diff
@@ -0,0 +1,367 @@
+```
+
+### Step 5: Alias Record
+
+Then, we can create an alias record that points to the CloudFront distribution that we configured. Here is the function for it.
+
+```typescript
+function createAliasRecord(
+    targetDomain: string, distribution: aws.cloudfront.Distribution): aws.route53.Record {
+    const domainParts = getDomainAndSubdomain(targetDomain);
+    const hostedZoneId = aws.route53.getZone({ name: domainParts.parentDomain }, { async: true }).then(zone => zone.zoneId);
+    return new aws.route53.Record(
+        targetDomain,
+        {
+            name: domainParts.subdomain,
+            zoneId: hostedZoneId,
+            type: "A",
+            aliases: [
+                {
+                    name: distribution.domainName,
+                    zoneId: distribution.hostedZoneId,
+                    evaluateTargetHealth: true,
+                },
+            ],
+        });
+}
+
+const aRecord = createAliasRecord(config.targetDomain, cdn);
+```
+
+---
+
+Now that we have finished implementing the infrastructure of deploying. After setting the AWS configuration we can run `pulumi up` to execute the Pulumi code to deploy this on AWS. However, there is a more convenient way to do so.
+
+## Github Workflow (Optional)
+
+Github Workflow adopts the Continuous Delivery/Integration concept and specifies a series of steps to run and deploy the program. We can take advantage of it with our CMS.
+
+Under the `.github/workflow` we have implemented the workflow that we described above. The sixth step "Assume Role" is for people who had an AWS IAM user access token and want to assume the role of admin in one of AWS account, then the code in `./scripts/assume-role.sh` contains a great function to assume a role in the AWS account.
+
+In the workflow, multiple Github Secret are referred to as `${{ secrets.ACCESS_TOKEN }}` where `ACCESS_TOKEN` is any Github secret. We can specify the secret inside the project setting of Github.
+
+![Github Secrets Settings](./github-secrets.jpg)
+
+`AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` is the IAM user tokens. `IAM_ROLE_ARN` and `EXTERNAL_ID` is the information for the role that we would like to assume. `PULUMI_ACCESS_TOKEN` is required for the automatic process and can be generated by clicking the blue button "New Access Token" on the Settings of [Pulumi console](https://app.pulumi.com)
+
+![Pulumi Tokens](./pulumi-token.jpg)
+
+---
+
+## Summary and Next Step
+
+Now we have deployed the stand-alone React CMS application on AWS. The [full code](https://github.com/pulumi/examples/tree/master/aws-ts-netlify-cms-and-oauth/cms) is included in [Pulumi's Example Repository](https://github.com/pulumi/examples).
+
+As mentioned previously, because we are deploying on AWS instead of Netlify, we have to found a way to substitute the Netlify Identity Service. Thus, the next step is to write an External OAuth Client-Server and deploy it on AWS as well. We would introduce that in our next blog post.
```

Comment:
```suggestion
The next article will show how to deploy on AWS instead of Netlify. To do this, we describe how to substitute the Netlify Identity Service by writing an External OAuth Client-Server and deploying it on AWS. Stay tuned for our next blog post.
```